scripts/release/mule/deploy/docker/docker.sh

```diff
@@ -23,24 +23,14 @@ pushd docker/releases
 
 if [ "$NETWORK" = mainnet ]
 then
-    # Build and push mainnet.
-   ./build_releases.sh
+  # Build and push mainnet.
+  ./build_releases.sh --tagname "$VERSION"
 
-    # Build and push testnet.
-   ./build_releases.sh --network testnet
-
-    if [ -z "$VERSION" ]
```

Comment:
This logic is unnecessary, because we test for this at the very start of the script.